dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.js

```diff
@@ -293,13 +293,21 @@ exports.getCompletedAmendmentByDealId = async (req, res) => {
  *  }
  */
 const findLatestCompletedAmendmentByFacilityId = async (facilityId) => {
+  const proceed = CONSTANTS.AMENDMENT.AMENDMENT_BANK_DECISION.PROCEED;
   if (ObjectId.isValid(facilityId)) {
     try {
       const collection = await db.getCollection('tfm-facilities');
       const amendment = await collection.aggregate([
         { $match: { _id: ObjectId(facilityId) } },
         { $unwind: '$amendments' },
-        { $match: { 'amendments.status': CONSTANTS.AMENDMENT.AMENDMENT_STATUS.COMPLETED } },
+        {
+          $match: {
+            $or: [
+              { 'amendments.status': CONSTANTS.AMENDMENT.AMENDMENT_STATUS.COMPLETED, 'amendments.submittedByPim': true, 'amendments.requireUkefApproval': false },
+              { 'amendments.status': CONSTANTS.AMENDMENT.AMENDMENT_STATUS.COMPLETED, 'amendments.bankDecision.decision': proceed, 'amendments.bankDecision.submitted': true }
```

Comment:
* De-construct as above.
* Do we need `proceed` variable, can it be de-constructed please?

Reply:
done